packages/atlas/atlas.config.yml

```diff
@@ -269,6 +269,13 @@ features:
   members:
     avatarServiceUrl: '$VITE_AVATAR_SERVICE_URL' # URL for avatar service - used to upload member's avatar
     hcaptchaSiteKey: '$VITE_HCAPTCHA_SITE_KEY' # Site key for hCaptcha - used to verify users are not bots when creating memberships - depends on hCaptcha being enabled in the faucet
+  walletConnect:
+    walletConnectProjectId: '$VITE_WALLET_CONNECT_PROJECT_ID' # WalletConnect project ID - used to connect to WalletConnect
+    metadata:
+      name: Atlas
+      description: Web3 video streaming platform
+      url: \#
```

Comment:
I think this should be full URL?
![IMG_C15FE4F79AC4-1](https://github.com/Joystream/atlas/assets/12646744/4ab4f81c-f904-4ceb-9e56-e20a60e23364)

Reply:
We could just populate it dynamically with the app URL?